ModularTegustation/tegu_overwrite_vars.dm

```diff
@@ -1,6 +1,6 @@
 
 /*
-		WELCOME TO THE FULPSTATION CODE Z-LEVEL!
+		WELCOME TO THE TEGUSTATION CODE Z-LEVEL!
```

Comment:
As said in the file - it is supposed to be at the very bottom of the code/ folder. Put it in its own file starting with "z", so it loads the last, effectively overwriting previous .dm files.